Scripts/Fuel Calc Time/test.yaml

```diff
@@ -0,0 +1,4 @@
+scripts: 
+  - fuel-calc-time.yolol
+cases: []

```

Comment:
If people keep doing this, I will have to add a code-coverage check to the CI...

Reply:
It's my fault for doing it with the ISAN one. That was a bit of a special case though.